apps/docs/components/CopyToClipboard/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useRef, useState } from "react";
+import CopyIcon from "./CopyIcon";
+import TickIcon from "./TickIcon";
+
+interface Props {
+  text: string;
+}
+
+export default function CopyToClipboard(props: Props) {
```

Comment:
Provide support for dynamic icon size.

<CopyToClipBoard size={24} text={code}/>
Keep some default size.

Reply:
Do you mean pass the size as prop (like tailwind classes), or make a size prop which has options like `small`, `medium` or `large` ?